src/Features/VisualBasic/Portable/ChangeSignature/VisualBasicChangeSignatureService.vb

```diff
@@ -618,12 +678,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ChangeSignature
             End Get
         End Property
 
-
-        Protected Overrides ReadOnly Property LanguageName As String
+        Protected Overrides ReadOnly Property SyntaxFacts As ISyntaxFacts
             Get
-                Return LanguageNames.VisualBasic
+                Return VisualBasicSyntaxFacts.Instance
             End Get
         End Property
 
+        Protected Overrides Function CreateExplicitParamsArrayFromIndividualArguments(newArguments As SeparatedSyntaxList(Of SyntaxNode), indexInExistingList As Integer, parameterSymbol As IParameterSymbol) As SyntaxNode
+            ' A params array cannot be introduced due to the addition of an omitted 
+            ' argument in VB because you cannot have a named argument to a params array.
+            Throw New InvalidOperationException()
+        End Function
+
+        Protected Overrides Function AddNameToArgument(newArgument As SyntaxNode, name As String) As SyntaxNode
+            Return DirectCast(newArgument, SimpleArgumentSyntax).WithNameColonEquals(NameColonEquals(IdentifierName(name)))
```

Comment:
what if it's not a SimpleArgSyntax?

Reply:
> OmittedArgumentSyntax 

Quite possibly . Though i'm fine with ChangeSig simply bailing out and not crashing on these.  but we need to test.

Reply:
Addressed. Thanks.